examples/CocoDataPyTorch.py

```diff
@@ -91,18 +95,38 @@ def generate_query(self, idx: int):
                 m = self.coco_detection.coco.annToMask(meta_info)
                 polygons = polygonFromMask(m)
                 adb_polygons = []
+                adb_boxes = []
                 for polygon in polygons:
                     adb_polygon = []
+                    left, top, right, bottom = 99999, 99999, 0, 0
```

Comment:
I guess 99999 is ok for unset for coco?

Reply:
i've not seen images of that resolution in coco